tests/unit/src/main/java/org/mule/tck/junit4/AbstractMuleContextTestCase.java

```diff
@@ -253,7 +257,7 @@ protected MuleContext createMuleContext() throws Exception {
         String workingDirectory = this.workingDirectory.getRoot().getAbsolutePath();
         LOGGER.info("Using working directory for test: " + workingDirectory);
         muleConfiguration.setWorkingDirectory(workingDirectory);
-        muleConfiguration.setId(this.getClass().getSimpleName() + "#" + name.getMethodName());
+        muleConfiguration.setId(serverId);
```

Comment:
I would change this for contextConfigurationId as this is a general test feature.